MonoGame.Framework/Media/SongCollection.cs

```diff
@@ -5,67 +5,115 @@
+        {
+            Dispose(true);
+            GC.SuppressFinalize(this);
+        }
+
+        /// <summary>
+        /// Performs application-defined tasks associated with freeing, releasing, or resetting unmanaged resources.
+        /// </summary>
+        /// <param name="disposing">Whether or not this song collection is disposing.</param>
+        protected virtual void Dispose(bool disposing)
+        {
+            if (disposed)
+                return;
+
+            if (disposing)
+            {
+                // Free any other managed objects here.
+                //
+            }
+
+            // Free any unmanaged objects here.
+            //
+            disposed = true;
         }
-		
-		public IEnumerator<Song> GetEnumerator()
+
+        /// <summary>Returns an enumerator that iterates through the song collection.</summary>
+        /// <returns>An enumerator that can be used to iterate through the song collection.</returns>
+        public IEnumerator<Song> GetEnumerator()
         {
             return innerlist.GetEnumerator();
         }
-		
+
         IEnumerator IEnumerable.GetEnumerator()
         {
             return innerlist.GetEnumerator();
         }
 
+        /// <summary>Gets the number of songs contained in the <see cref="SongCollection" />.</summary>
+        /// <returns>The number of songs contained in the <see cref="SongCollection" />.</returns>
         public int Count
         {
             get
             {
-				return innerlist.Count;
+                return innerlist.Count;
             }
         }
-		
-		public bool IsReadOnly
+
+        /// <summary>Gets a value indicating whether the <see cref="SongCollection" /> is read-only.</summary>
+        /// <returns>true if the <see cref="SongCollection" /> is read-only; otherwise, false.</returns>
+        public bool IsReadOnly
         {
-		    get
-		    {
-		        return this.isReadOnly;
-		    }
+            get
+            {
+                return this.isReadOnly;
+            }
         }
 
+
+        /// <summary>
+        /// Retrieves the song at the specified zero-based
+        /// index of this SongCollection.
+        /// </summary>
+        /// <param name="index">The index of the song</param>
+        /// <returns>the song at the specified index</returns>
         public Song this[int index]
         {
             get
             {
-				return this.innerlist[index];
+                return this.innerlist[index];
             }
         }
-		
-		public void Add(Song item)
+
+        /// <summary>Adds an item to the <see cref="SongCollection" />.</summary>
+        /// <param name="item">The object to add to the <see cref="SongCollection" />.</param>
+        /// <exception cref="ArgumentNullException">The song is null.</exception>
+        public void Add(Song item)
         {
 
             if (item == null)
-                throw new ArgumentNullException();
+                throw new ArgumentNullException(nameof(item));

```

Comment:
The build bot uses C# 5 for console compatibility, so this will have to be replaced with "item."

Reply:
Yup.